.travis.yml

```diff
@@ -5,3 +5,4 @@ addons:
     apt:
         packages:
             - gstreamer1.0-plugins-good
+            - gstreamer1.0-plugins-bad
```

Comment:
I just need these plugins not for production but just for tests. There is [timecodestamper](https://gstreamer.freedesktop.org/documentation/timecode/timecodestamper.html?gi-language=c) which can be used for testing timecode metadata. But true is that `timecodestamper` is available since 1.10.

Reply:
Yes, just wondering if there's any way to do test differently without this element?  If we can keep the test requirements to plugins-good, I'd be happier.